mesos-docker/run/run.sh

```diff
@@ -337,6 +341,14 @@ function start_slaves {
       docker exec "$SLAVE_CONTAINER_NAME"_"$i" /usr/local/sbin/hadoop-daemon.sh --script hdfs start datanode
     fi
 
+    if [[ -n $START_SHUFFLE_SERVICE ]]; then
+      start_shuffle_service_command="/bin/mkdir -p $SPARK_CONF_FOLDER"
+      start_shuffle_service_command="$start_shuffle_service_command; /bin/echo 'spark.shuffle.service.port $((7336 + i))' >> $SPARK_CONF_FOLDER/spark-defaults.conf"
+      start_shuffle_service_command="$start_shuffle_service_command; /bin/tar -C /opt/ -xf /var/spark/$SPARK_FILE"
```

Comment:
I'd like for us to eventually use `spark.mesos.executor.home`, to make staging faster (no need to copy and unzip the tar ball in each Docker container. Actually, until now, it worked fine to mount a directory instead of a file inside Docker. Maybe the untar step could be optional?